Ivan_Shytskyi/Lesson_7/Lesson_7.Homework/Program.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Globalization;
+using System.Net.Security;
+using System.Reflection.Metadata.Ecma335;
+
+Console.WriteLine("Homework");
+bool Compare(string str1, string str2)
+{
+    if (str1 == str2) return true;
+    else return false;
+}
+
+bool Compare_1(string str1, string str2)
+{
+    if (str1.Length == str2.Length) return true;
+    else return false;
+}
+
+static (int letter, int digit, int anather) Analyze(string str)
+{
+    int letter = 0;
+    int digit = 0;
+    int anather = 0; 
+    var arr = str.ToCharArray();
+    for (int i = 0; i < arr.Length; i++)
+    {
+        if (char.IsLetter(arr[i]))
+            letter++;
+        else if (char.IsDigit(arr[i]))
+            digit++;
+        else
+            anather++;
+    }
+    return (letter, digit, anather);
+}
+
+string Sort(string str)
+{
+    str = str.ToLower();
+    var arr = str.ToCharArray();
+    for (int i = 1; i < arr.Length; i++)
+    {
+        for (int j = 0; j < arr.Length - i; j++)
+        {
+            if (arr[j] > arr[j + 1])
+            {
+                char tmp = arr[j + 1];
+                arr[j + 1] = arr[j];
+                arr[j] = tmp;
+            }
+        }
+    }
+    return new string(arr);
+}
+
+static char[] Duplicate(string str)
+{
+    string d = "";
+    for (int i = 0; i < str.Length; i++)
+    {
+        d += str[i];
```

Comment:
??? method to copy string?
Read the task carefully:
Duplicate that will return array of characters that are duplicated in input string (e.g. 'Hello and hi' -> ['h', 'l'])